core/src/main/java/nu/yona/server/device/service/DeviceAnonymizedDto.java

```diff
@@ -61,4 +63,9 @@ public String getVpnLoginId()
 	{
 		return Optional.ofNullable(firebaseInstanceId);
 	}
+
+	public Locale getLocale()
+	{
+		return Translator.EN_US_LOCALE; // TODO: Take this from the entity
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://10.127.10.107:9000/coding_rules#rule_key=squid%3AS1135)